MessageBird/Objects/Request.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Text;
+
+namespace MessageBird.Objects
+{
+    public class Request
```

Comment:
Perhaps add a comment here to clarify what the class is used for. Out of context, it is not clear from the code that it's used for (request) signing.